pkg/storage/export/virt-exportserver/exportserver.go

```diff
@@ -140,6 +163,11 @@ func (s *exportServer) getHandlerMap(vi VolumeInfo) map[string]http.Handler {
 		result[vi.RawGzURI] = s.GzipHandler(p)
 	}
 
+	if vi.VMURI != "" {
+		result[vi.VMURI] = s.VmHandler(p, s.Volumes)
+		result[filepath.Join(vi.VMURI, "secret")] = s.TokenSecretHandler(s.TokenGetter)
```

Comment:
Changed the URIs to be `internal|external/manifests/all` for all the manifests minus the secret, and `internal|external/manifests/secret` for the CDI compatible secret. Defaulted to JSON output, and if you pass `Accept:application/yaml` header, it will return yaml instead.

No parameters at this point in time, will likely do that in a follow up PR.